src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class ColorSupplier {
+    public static String getRandomColor() {
+        int randNumber = new Random().nextInt(Color.values().length);
```

Comment:
Magic numbers should be avoided. Consider declaring a constant for the length of the `Color` enum values.